tests/test_other.py

```diff
@@ -7951,6 +7951,15 @@ def test_closure_externs(self):
                       '--pre-js', test_file('test_closure_externs_pre_js.js'),
                       '--closure-args', '--externs "local_externs.js"'])
 
+  def test_closure_externs_one_arg(self):
+    # Test that if an externs file is specified with --externs=foo.js (e.g. when using Bazel),
+    # its absolute path is still used (otherwise the file will not be found).
```

Comment:
How about moving this into a loop in in the above test:

```
for arg in ['--externs "local_externs.js"', '--externs=local_externs.js']:
   self.run_process...
```

Reply:
Done.